src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -8,6 +8,7 @@
 using System.Runtime.Serialization;
 
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.ComponentModel.Component))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.ComponentModel.ISupportInitialize))]
```

Comment:
This isn't really necessary given we haven't ship this contract stable but it shouldn't hurt anything either.